packages/ketcher-core/src/application/render/restruct/rebond.ts

```diff
@@ -57,18 +57,167 @@ class ReBond extends ReObject {
     return true
   }
 
+  static bondRecalc(bond: ReBond, restruct: ReStruct, options: any): void {
```

Comment:
This function is just converted from member to static function, b/c we need to recalculate bond positions after they moved (it is used later in dropAndMerge.ts)